packages/vite/src/node/config.ts

```diff
@@ -1119,6 +1129,14 @@ async function bundleConfigFile(
               false,
             )?.id
           }
+          const isTsConfigPathImport = (id: string): boolean => {
+            if (!tsConfigPaths.length) {
+              return false
+            }
```

Comment:
Since **Array.prototype.some** always returns false when called on an empty array, the tsConfigPaths.length check is unnecessary.